src/net/sourceforge/kolmafia/KoLmafia.java

```diff
@@ -1861,15 +1874,79 @@ public static String whoisPlayer(final String player) {
   public static boolean isPlayerOnline(final String player) {
     // This player is currently online.
     // This player is currently online in channel clan.
-    // This player is currently away from KoL in channel trade and listening to clan.
+    // This player is currently away from KoL in channel trade and listening to
+    // clan.
     String text = KoLmafia.whoisPlayer(player);
     return text != null && text.contains("This player is currently");
   }
 
   private static class UpdateCheckRunnable implements Runnable {
     public void run() {
-      // TODO: Check for new version on jenkins\github after migration is complete. See revision
-      // history for old release update check.
+      boolean checkUpdate = Preferences.getBoolean("checkRssUpdate");
+      if (!checkUpdate) {
+        return;
+      }
+
+      long lastUpdate = Long.parseLong(Preferences.getString("lastRssUpdate"));
```

Comment:
Why not just Preferences.getLong?

Reply:
this is an existing preference, I didn't want to change it from a string